content-repo/extra-docs/releases/20.8.1.md

```diff
@@ -0,0 +1,576 @@
+# Cortex XSOAR Content Release Notes for version 20.8.1 (88914)
+##### Published on 16 August 2020
+
+---
+
+Welcome to the 20.8.1 Content Release for Cortex XSOAR. Starting from the 20.6.0 release, we restructured our release notes to be based on **Content Packs**. One of our team's top priorities is making our Content more accessible and understandable for both users and contributors. In this effort, we recently moved our [Content repo](https://github.com/demisto/content) to work in Pack format, in which there is a clear separation and grouping of Content artifacts. Each Content Pack provides a clear grouping of related Content artifacts used to either implement a use case, implement an integration or provide a clear set of functionality. Our new release notes are structured around Content Packs and you will see related Content artifacts grouped together according to Packs. We hope you will find this new format useful and clear.
+
+#### For Cortex XSOAR version 5.5 and earlier, you can still install content updates directly in the platform. 
+
```

Comment:
@glicht do we want to add here a comment about the content packs zip ?

Reply:
Don't think there is need. As it will be documented here: https://github.com/demisto/etc/issues/27288. I've also just added a commit to include the link to the packs and specify xsoar version.